Pod/Classes/RxRealm.swift

```diff
@@ -450,17 +437,16 @@ extension Reactive where Base: Realm {
      - parameter: onError - closure to implement custom error handling
      - returns: `AnyObserver<S>`, which you can use to subscribe an `Observable` to
      */
-    public static func delete<S: Sequence>(onError: ((S?, Error)->Void)? = nil)
-        -> AnyObserver<S>  where S.Iterator.Element: Object {
-
+    public static func delete<S: Sequence>(onError: ((S?, Error) -> Void)? = nil)
+        -> AnyObserver<S> where S.Iterator.Element: Object {
         return AnyObserver { event in
 
             guard let elements = event.element,
                 var generator = elements.makeIterator() as S.Iterator?,
                 let first = generator.next(),
                 let realm = first.realm else {
-                    onError?(nil, RxRealmError.unknown)
-                    return
+                onError?(nil, RxRealmError.unknown)
```

Comment:
If this is aligned this way I think you should align the `var` and `let`s to the first (`let element`)